instance/tests/integration/integration_instance.py

```diff
@@ -235,7 +235,7 @@ def assert_theme_provisioned(self, instance, appserver, application):
 
         # The logo is found in a line like this in the HTML:
         #       <img src="/static/simple-theme/images/logo.82fb8d18479f.png" alt="danieltest1b Home Page"/>
-        logo_extractor = re.search(r'<img src="(/static/simple-theme/images/logo.[a-z0-9]+\.png)" ',
+        logo_extractor = re.search(r'<img\s+class="logo"\s+src="(/static/simple-theme/images/logo.[a-z0-9]+\.png)"',
```

Comment:
Nit: I didn't noticed that you changed the regex here, so if you want, you can change the formatting back.

Reply:
That's ok, this is the same format used in the rest of the module.